kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -114,7 +115,7 @@ case class KyuubiConf(loadSysDefault: Boolean = true) extends Logging {
 
   /** Get all parameters as map */
```

Comment:
Please update the comment to mention the ordered. And better to add a simple UT to ensure it works as expected.

Reply:
UT added and method comment updated.